### PR TITLE
renames base project to default project in code and copy

### PR DESCRIPTION
**Pull request by dimitropoulos**
During standup today we discussed renaming the "base project" to "default project" to be more clear to users what it is and what it does.  This PR makes that change.  Individual commit notes are in the commit descriptions.

The main places this changes for users are in the tooltip over the icon for the default space.

before:
Screenshot_20210824_171348

after:
Screenshot_20210824_171248

### CODE REVIEW

File: packages/insomnia-app/app/ui/components/dropdowns/response-history-dropdown.tsx [hunk 1]
```diff
@@ -158,7 +159,7 @@ class ResponseHistoryDropdown extends PureComponent<Props> {
       activeEnvironment,
       ...extraProps
     } = this.props;
-    const environmentName = activeEnvironment ? activeEnvironment.name : 'Base';
+    const environmentName = activeEnvironment ? activeEnvironment.name : strings.defaultProject.singular;
```
Comment (dimitropoulos) [inline packages/insomnia-app/app/ui/components/dropdowns/response-history-dropdown.tsx]: so I'm doing a self review and this suddenly seems wrong.  can anyone confirm?

Comment (reynolek) [reply packages/insomnia-app/app/ui/components/dropdowns/response-history-dropdown.tsx]: This seems to be changing the environment name, so thats probably wrong.

Comment (develohpanda) [reply packages/insomnia-app/app/ui/components/dropdowns/response-history-dropdown.tsx]: Good catch, this shouldn't change

File: packages/insomnia-app/app/models/project.ts [hunk 2]
```diff
@@ -9,13 +9,13 @@ export const prefix = 'proj';
 export const canDuplicate = false;
 export const canSync = false;
 
-export const BASE_PROJECT_ID = `${prefix}_base-project`;
+export const DEFAULT_PROJECT_ID = `${prefix}_base-project`;
```
Comment (develohpanda) [inline packages/insomnia-app/app/models/project.ts]:
```suggestion
export const DEFAULT_PROJECT_ID = `${prefix}_default-project`;
```

File: packages/insomnia-app/app/ui/components/dropdowns/response-history-dropdown.tsx (hunk 1 above)
Comment (develohpanda) [reply packages/insomnia-app/app/ui/components/dropdowns/response-history-dropdown.tsx]: Reverted in [`6015e83` (#3939)](https://github.com/Kong/insomnia/pull/3939/commits/6015e83ec33ef805560b68f9570af78ff589a8b5)

**Review by develohpanda — APPROVED** [review]
LGTM

image

image

**Pull request merged**